recommenders/evaluation/python_evaluation.py

```diff
@@ -696,46 +698,131 @@ def get_top_k_items(
 }
 
 # diversity metrics
-class PythonDiversityEvaluation:
-    """Python Diversity Evaluator"""
+def check_column_dtypes_diversity_serendipity(func):
```

Comment:
Shouldn't we have this decorator private (with an '_' at the front of the name)?

Reply:
We did not use "_" in existing code, e.g. the function "check_column_dtypes" does not have "_" in front of the function name. Therefore I don't add "_" to be consistent.

Reply:
No, I meant adding underscore (_), not quotes. Currently you see these methods in [readthedocs](https://microsoft-recommenders.readthedocs.io/en/staging/evaluation.html#recommenders.evaluation.python_evaluation.check_column_dtypes), where they are probably not needed.

Reply:
There is another issue with the docstrings. The args are missing in the python functions e.g. [here](https://microsoft-recommenders.readthedocs.io/en/staging/evaluation.html#recommenders.evaluation.python_evaluation.serendipity) because before they were inside the encapsulating class but now they are required.